### PR TITLE
Fix incorrect docstring for promote-certain-elements-to-title feature

### PR DESCRIPTION
**Pull request by MarkLindblad**

### CODE REVIEW

File: lib/sycamore/sycamore/transforms/partition.py [hunk 1]
```diff
@@ -419,10 +419,15 @@ class ArynPartitioner(Partitioner):
...
-            default: {"promote_title": True ,  "title_candidate_elements":["Section-header", "Caption"]}
+        output_label_options: A dictionary for configuring output label behavior. It supports two options:
+            promote_title, a boolean specifying whether to pick the largest element by font size on the first page
+                from among the elements that are of a type specified in title_candidate_elements and promote it to
+                type Title if there is no element on the first page of type Title already.
+            title_candidate_elements, a list of strings representing the label types allowed to be promoted to
+                a title.
+            Here is an example set of output label options:
+                {"promote_title": True, "title_candidate_elements": ["Section-header", "Caption"]}
+            default: None (no elements are promoted to titles)
```
Comment (karanataryn) [inline lib/sycamore/sycamore/transforms/partition.py:430]: Default- promote_title: False? None seems harder to grok.

Comment (MarkLindblad) [reply lib/sycamore/sycamore/transforms/partition.py:430]: Are you suggesting we change the default value of `output_label_options`? It's default value is currently `None` - which is _effectively_ {"promote_title": False}, but we error when promote_title is specified and title_candidate_elements is not specified, and the implementation actually uses a `None`, not a {"promote_title": False}.

**Review by karanataryn — APPROVED** [review]
Approved with suggestions.

File: lib/sycamore/sycamore/transforms/partition.py [hunk 2]
```diff
@@ -419,10 +419,15 @@ class ArynPartitioner(Partitioner):
...
              which can be set to 'boxes' or 'lines' to control the granularity of output.
         source: The application that is using the partitioner. This is used for logging purposes.
-        output_label_options: A dictionary for configuring output label behavior. It supports two options: 
-        promote_title, a boolean that specifies whether to add a title to partitioned elements if one is missing, and
-            title_candidate_elements, a list of strings representing labels for potential titles.
-            default: {"promote_title": True ,  "title_candidate_elements":["Section-header", "Caption"]}
+        output_label_options: A dictionary for configuring output label behavior. It supports two options:
+            promote_title, a boolean specifying whether to pick the largest element by font size on the first page
+                from among the elements that are of a type specified in title_candidate_elements and promote it to
+                type Title if there is no element on the first page of type Title already.
```
[the remark this answers is not in the document]
Comment (MarkLindblad) [reply lib/sycamore/sycamore/transforms/partition.py]: Done.

File: lib/sycamore/sycamore/transforms/partition.py [hunk 3]
```diff
@@ -419,10 +419,15 @@ class ArynPartitioner(Partitioner):
              either pdfminer or OCR. Currently supports the 'object_type' property for pdfminer,
              which can be set to 'boxes' or 'lines' to control the granularity of output.
         source: The application that is using the partitioner. This is used for logging purposes.
-        output_label_options: A dictionary for configuring output label behavior. It supports two options: 
-        promote_title, a boolean that specifies whether to add a title to partitioned elements if one is missing, and
-            title_candidate_elements, a list of strings representing labels for potential titles.
-            default: {"promote_title": True ,  "title_candidate_elements":["Section-header", "Caption"]}
+        output_label_options: A dictionary for configuring output label behavior. It supports two options:
+            promote_title, a boolean specifying whether to pick the largest element by font size on the first page
+                from among the elements that are of a type specified in title_candidate_elements and promote it to
```
[the remark this answers is not in the document]
Comment (MarkLindblad) [reply lib/sycamore/sycamore/transforms/partition.py]: Done.

File: lib/sycamore/sycamore/transforms/partition.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (MarkLindblad) [reply lib/sycamore/sycamore/transforms/partition.py]: Done.

**Pull request merged**